worlds/kdl3/options.py

```diff
@@ -274,7 +284,8 @@ class KirbyFlavorPreset(Choice):
     option_orange = 11
     option_lime = 12
     option_lavender = 13
-    option_custom = 14
+    option_miku = 14
+    option_custom = 15
```

Comment:
You have apparently run into the issue that you want "Custom" to be last, yet need to add presets.
You might run into it again. Why not give custom a higher value, and leave room for next time, so that you don't need to do such a breaking change next time?

Reply:
Also, you can totally just

```
option_a: 1
option_b: 3
option_c: 2
```

Reply:
Personally, I don't really mind making this type of change. It's decently rare (no one has actually asked to contribute any palettes, I just made this one for fun), and I much prefer keeping them in order.